bootstrap.py

```diff
@@ -1334,7 +1335,7 @@ def exec_service(service, command, failonerror=True):
         # >                  optionally clean and install Puppet agent
         # >                  optionally remove legacy RHN packages
         if 'katello-agent' not in options.skip:
```

Comment:
The skip option should not be `katello-agent` now anymore, right? :)

Reply:
Yeah, I debated this. Changing this breaks a working config (`--skip katello-agent` now doesn't work).

Reply:
I'd say leave `katello-agent` in the skip list, but make it basically a NOOP, as this is the default now. 

And add `k-h-t` as a new skip item. Or maybe don't allow skipping it at all?

Reply:
If we don't allow skipping `k-h-t` then we'd just get a PR for it in a few weeks anyway :)

Reply:
@sideangleside circling back to this. I still think skipping `katello-agent` should be a NOOP, and `k-h-t` should get an own skip step.

after that, I am good with merging this